Toshi/Source/Toshi/Render/TRender.h

```diff
@@ -133,7 +133,7 @@ namespace Toshi
 		TMatrix44 m_oModelWorldMatrix;          // 0x00D0
 		TMatrix44 m_oViewWorldMatrix;           // 0x0110
 		// ...
-		TMatrix44 m_oWorldModelMatrix;          // 0x036C
+		TMatrix44 m_oWorldModelMatrix;          // 0x032�
```

Comment:
Whoops

Reply:
oops I think it was russian letter (we have a key on keyboard that prints C and C in both language layout but they are actually different characters). will fix that in the next commit